compiler/tests/schema/generic_service.bond

```diff
@@ -7,3 +7,7 @@ service Foo<Payload>
     Payload foo33(Payload);
 }
 
+service Bar<Payload> : Foo<Payload>
```

Comment:
As I mentioned above, I'd prefer to see a different .bond file used for the experimental syntax.